_includes/v2.2/app/sqlalchemy-basic-sample.py

```diff
@@ -1,38 +1,111 @@
-from __future__ import print_function
+import random
+from math import floor
 from sqlalchemy import create_engine, Column, Integer
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
+from cockroachdb.sqlalchemy import run_transaction
 
 Base = declarative_base()
 
+
 # The Account class corresponds to the "accounts" database table.
 class Account(Base):
     __tablename__ = 'accounts'
     id = Column(Integer, primary_key=True)
     balance = Column(Integer)
 
-# Create an engine to communicate with the database. The "cockroachdb://" prefix
-# for the engine URL indicates that we are connecting to CockroachDB.
-engine = create_engine('cockroachdb://maxroach@localhost:26257/bank',
-                       connect_args = {
-                           'sslmode' : 'require',
-                           'sslrootcert': 'certs/ca.crt',
-                           'sslkey':'certs/client.maxroach.key',
-                           'sslcert':'certs/client.maxroach.crt'
-                       })
-Session = sessionmaker(bind=engine)
+
+# Create an engine to communicate with the database. The
+# "cockroachdb://" prefix for the engine URL indicates that we are
+# connecting to CockroachDB using the 'cockroachdb-python' dialect.
+# For more information, see
+# https://github.com/cockroachdb/cockroachdb-python.
+
+engine = create_engine(
+    'cockroachdb://maxroach@localhost:26257/bank',
+    connect_args={
```

Comment:
Instead of having two files that differ only in this block, I'd suggest just having one with some sort of `if secure:` check around the create_engine call.

Reply:
Fixed by adding some conditional code to this sample and then deleting the "insecure" example file.